vulcan/lib/vulcan.rb

```diff
@@ -14,4 +14,34 @@ def setup_db(load_models = true)
 
     require_relative 'models' if load_models
   end
+
+  def setup_yabeda
+    Yabeda.configure do
+      group :vulcan do
+        histogram :job_runtime do
+          comment "Time spent by each cell, including storage and docker execution."
+          unit :seconds
+          buckets [5, 15, 60, 150, 300]
+        end
+
+        gauge :storage_disk_usage do
+          comment "Amount (in bytes) used by vulcan storage directories"
+          tags [:dir]
+        end
+      end
+
+      collect do
+        output = `du #{Vulcan.instance.config(:data_folder)} --max-depth 1` rescue ""
```

Comment:
An example of using a yabeda job to periodically collect data as a metric.  Here I periodically collect disk usage in the data folder and set metrics of that disk usage.  You can do anything you'd like in these blocks and have them safely run while the webserver is up.